doc/release/RELEASE-NOTES.md

```diff
@@ -65,6 +65,7 @@ to modify moderation event `status`.
 * The RDS instance for `production` has been upgraded to `db.m6in.4xlarge` and configured to operate in a single Availability Zone.
 
 ### Bugfix
+* [OSDEV-1388](https://opensupplyhub.atlassian.net/browse/OSDEV-1388) - The waiter from boto3 cannot wait more than half an hour so we replaced it with our own.
```

Comment:
Please create a new section for release version 1.26 and add this information to it. You can use the template from the RELEASE-NOTES-TEMPLATE.md file.

Reply:
done